README.md

```diff
@@ -1,28 +1,101 @@
+
+Finally, you can build the project via Gradle:
+
+```bash
+$ ./gradlew assembleDebug
+```
+
+### How to Build the Production Sample App
+
+First you will need to create the keystore file at `testapp/release-keystore.jks`. Next you must define environment variables for the keystore settings.
+
+```
+MINIAPP_RELEASE_KEY_ALIAS=my_alias
+MINIAPP_RELEASE_KEY_PASSWORD=my_password
+MINIAPP_KEYSTORE_PASSWORD=my_keystore_password
+```
+
+Next, you must define the prod settings for the Sample App as either environment variables are Gradle Properties.
+
+```
+MINIAPP_PROD_SERVER_BASE_URL=https://www.example.com/
+HOST_APP_PROD_ID=test-host-app-id
+HOST_APP_PROD_SUBSCRIPTION_KEY=test-subs-key
+HOST_APP_PROD_UA_INFO=MiniAppDemoApp/1.0.0
+HOST_APP_PROD_VERSION=test-host-app-version
+```
+
+Finally, you can build the release config for the Sample App.
+
+```bash
+$ ./gradlew assembleRelease
+```
+
+## How to Test the Sample App
+
+We currently don't provide an API for public use, so you must provide your own API.
+
+## Continuous Integration and Deployment
+
+[CircleCI](https://circleci.com/gh/rakutentech/android-miniapp) is used for building and testing the project for every pull request. It is also used for publishing the SDK and the Sample App. 
+
+We use jobs from two CircleCI Orbs (see the [android-buildconfig](https://github.com/rakutentech/android-buildconfig/tree/master/circleci) repo): [android-sdk Orb](https://github.com/rakutentech/android-buildconfig/blob/master/circleci/android-sdk/README.md) and [app-center Orb](https://github.com/rakutentech/android-buildconfig/blob/master/circleci/app-center/README.md). See the Readme for those Orbs for more information on what the jobs do.
+
+### Merge to Master
+
+The following describes the steps CircleCI performs when a branch is merged to master.
+
+1. We trigger CircleCI by merging a branch to master.
+2. CI builds the SDK and Sample App, run tests, linting, etc.
+3. CI publishes staging build of Sample App to App Center "Testers" group.
+
+### Release
+
+The following describes the steps CircleCI performs to release a new version of the SDK.
+
+1. We trigger CircleCI by pushing a git tag to this repo which is in the format `vX.X.X`.
+2. CI builds SDK and Sample App, run tests, linting, etc.
+3. CI pauses the workflow for verification from user.
+4. After approval, CI publishes the SDK to [Bintray](https://bintray.com/ssed-oss-jcenter/ssed-mobile-libs/android-miniapp).
+5. CI publishes generated documentation to [Github Pages site](https://rakutentech.github.io/android-miniapp/).
+6. CI publishes production build of Sample App to App Center's "Production" group.
+
+*Note:* It's also possible to publish a snapshot version of the SDK by using a `-` in the version name, such as `v1.0.0-alpha` (and the generated version name will be `1.0.0-SNAPSHOT`). In this case, the snapshot will be published to the [JFrog OSS Snapshot repository](https://oss.jfrog.org/) and the documentation publishing will be skipped.
```

Comment:
Do we need to mention jfrog url as https://oss.jfrog.org/webapp/#/artifacts/browse/tree/General/oss-snapshot-local/com/rakuten/tech/mobile/miniapp/miniapp instead of https://oss.jfrog.org/ 🤔